rabbitmq/docker-entrypoint.sh

```diff
@@ -0,0 +1,404 @@
+#!/bin/bash
```

Comment:
Given this is 400+ lines, we need a license for this script.

Reply:
I copied docker-entrypoint.sh and changed lines of "gosu" from https://github.com/docker-library/rabbitmq/tree/master/3.7/ubuntu. 
So I need copy https://github.com/docker-library/rabbitmq/blob/master/LICENSE to this rabbmitmq folder?